### ISSUE TITLE
Unknown column glpi_profilerights which makes front/profile.php empty

### ISSUE THREAD

**Issue by lirmikey** [glpi-project/glpi#974]
In version 0.90.3 we get this error. Have searched in search.class and profile.class but do not understand how it works:

```
*** MySQL query error:
  SQL: SELECT \'deleted for security\' AS currentuser,
                         `glpi_profiles`.`name` AS `ITEM_0`,
                        `glpi_profiles`.`id` AS `ITEM_0_id`,
                        
                        `glpi_profiles`.`interface` AS `ITEM_1`,  `glpi_profiles`.`is_default` AS `ITEM_2`,  `glpi_profiles`.`date_mod` AS `ITEM_3`,  `glpi_profilerights_aa95ab282e3ad953cfb2b7eafa455ecb`.`right` AS `ITEM_4`,  `glpi_profiles`.`id` AS id  FROM `glpi_profiles` LEFT JOIN `glpi_profilerights`  AS `glpi_profilerights_aa95ab282e3ad953cfb2b7eafa455ecb`
                                             ON (`glpi_profiles`.`id` = `glpi_profilerights_aa95ab282e3ad953cfb2b7eafa455ecb`.`profiles_id`
                                                 AND `glpi_profilerights_aa95ab282e3ad953cfb2b7eafa455ecb`.`name`= \'password_update\' ) ORDER BY ITEM_4 DESC  LIMIT 0, 200
  Error: Unknown column 'glpi_profilerights_aa95ab282e3ad953cfb2b7eafa455ecb.right' in 'field list'
  Backtrace :
  inc/search.class.php:905                           
  inc/search.class.php:100                           Search::constructDatas()
  inc/search.class.php:81                            Search::showList()
  front/profile.php:44                               Search::show()
```

**Comment by orthagh** [comment]
Hello @lirmikey 

Here is the fix (pushed today): https://github.com/glpi-project/glpi/commit/86e0e8b#diff-2e194c2ff8eaf1f6d72247482dbd6cceL2024

For reference: #962

**Comment by ddurieux** [comment]
It was difficult to backend all search engine fixes in 0.90.x, but the right field we can
